bofire/data_models/strategies/api.py

```diff
@@ -2,6 +2,7 @@
 
 from bofire.data_models.strategies.doe import DoEStrategy
 from bofire.data_models.strategies.factorial import FactorialStrategy
+from bofire.data_models.strategies.polytope import PolytopeSampler
```

Comment:
Since there are now also strategies that are called `Sampler` the naming becomes somewhat inconsistent, no? `RandomStrategy` is also just a sampling.

Reply:
Fully agree, but I do not have a proper solution, how would you do the naming?

Reply:
Hmm... The easiest would be that all are strategies. Which they are. With Samplers we probably mean not-predictive, right? Then we would call DoE a sampler and Random also a sampler. Depends on what sampler actually means.

Reply:
Yep, it means non-predictive. And something like `PolytopeStrategy` sounds weird ..., I think I will merge this PR first and we create an issue to solve this naming problem, ok? Or perhaps just schedule a meeting ...